tests/shared/utils.py

```diff
@@ -121,7 +121,10 @@ def transfer(self, node_id: str, data: bytes, port: int = 8081, timeout: int | N
         LOGGER.debug(f"POST Transfer file to {node_id} ({self.container.name} - {self.node_id})")
         port = self.__external_port(port)
         response: requests.Response = requests.post(
-            f"http://localhost:{port}/transfer-file/{node_id}", data, headers=http_client_headers, timeout=timeout
+            f"http://localhost:{port}/transfer-file/reliable/{node_id}",
```

Comment:
Same,
New arg `transport: str = "reliable"` +
and then `f"http://localhost:{port}/transfer-file/{transport}/{node_id}"`
?